index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+
+    <meta charset="UTF-8">
+
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="style.css">
+
+    <title>Dr. Norman Borlaug</title>
+
+</head>
+
+<body>
+
+    <main class="main">
+
+        <p class="main-title">Dr. Norman Borlaug</p>
+
+        <p class="sub-title">The man who saved a billion lives</p>
+
```

Comment:
Good job on html 👏 Please consider using HTML semantic tag, instead of using paragraph  tag for the main title use h1
because it is important for search engine optimization